### PR TITLE
Fix `declaration-block-no-duplicate-properties` autofix for `!important`

### PR DESCRIPTION
**Pull request by sidx1024**
> Which issue, if any, is this issue related to?

Fixes https://github.com/stylelint/stylelint/issues/6523

> Is there anything in the PR that needs further explanation?

No, that's it.

### CODE REVIEW

**Review by ybiquitous — CHANGES_REQUESTED** [review]
@sidx1024 Thanks for creating the PR! Additionally, could you add test cases to `testRule` with other secondary options?

- `ignore: [ 'consecutive-duplicates-with-different-values' ]`
- `ignore: [ 'consecutive-duplicates-with-same-prefixless-values' ]`
- `ignoreProperties: [ 'color' ]`

**Comment by sidx1024** [comment]
@ybiquitous Added the requested test cases.

File: lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js [hunk 1]
```diff
@@ -291,6 +324,11 @@ testRule({
 			fixed: 'p { width: -moz-fit-content; }',
 			message: messages.rejected('width'),
 		},
+		{
+			code: 'p { width: -moz-fit-content; width: -moz-fit-content !important; }',
+			fixed: 'p { width: -moz-fit-content !important; }',
+			message: messages.rejected('width'),
+		},
```
Comment (ybiquitous) [inline lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js:331]: [suggest] To cover the following line,

```
lib/rules/declaration-block-no-duplicate-properties/index.js#L125

Added line #L125 was not covered by tests
```

could you please add a more test case?

```suggestion
		},
		{
			code: 'p { width: 100% !important; width: -moz-fit-content; }',
			fixed: 'p { width: 100% !important; }',
			message: messages.rejected('width'),
		},
```

**Review by ybiquitous — CHANGES_REQUESTED** [review]
@sidx1024 Thank you. Could you address one more suggestion, please?

**Review by ybiquitous — APPROVED** [review]
Thanks. LGTM 👍🏼

**Pull request merged**